test/unit/test-layout.cxx

```diff
@@ -105,12 +102,12 @@ TEST(LayoutTest, View)
 {
   int* data = new int[10];
 
-  using layout = RAJA::OffsetLayout<int, RAJA::PERM_I, int>;
+  using layout = RAJA::OffsetLayout<>;
 
   /*
    * View is constructed by passing in the layout.
    */
-  RAJA::View<int, layout> view(data, layout({1}, {10}));
+  RAJA::View<int, layout> view(data, layout({1}, std::array<int, 1>{10}));
```

Comment:
is the `std::array` absolutely required here? When I was implementing this originally I tried really hard to avoid it